lib/test.js

```diff
@@ -58,8 +58,16 @@ function configure(sinon, config) {
             } catch (e) {
                 exception = e;
             }
-
-            if (typeof oldDone !== "function") {
```

Comment:
Ah, someone shoot me. I actually did not see that you added the comment until after my comment. LGTM!